CODE_OF_CONDUCT.md

```diff
@@ -1,9 +0,0 @@
-The Ember team and community are committed to everyone having a safe and inclusive experience.
```

Comment:
please restore this or update with one recommended by adopted-ember-addons

Reply:
I've restored it for now, but will update it ASAP.